src/app/portlets/dot-edit-page/content/services/html/iframe-edit-mode.js.ts

```diff
@@ -5,6 +5,33 @@ export const EDIT_PAGE_JS = `
 (function () {
     var forbiddenTarget;
 
+    function replaceStyles(styles, newStyle, elem) {
```

Comment:
Why do we need this? can we just overwrite the style properties we have to for each state?